src/Assignment_1/Constants.java

```diff
@@ -1,11 +1,17 @@
 package Assignment_1;
 
+/**
+* This class is made to avoid the use of repeated value 
+* which is use often
+ */
+
 public class Constants{
-	public final static String quantity = "quantity";
-    public final static String type = "type";
-    public final static String name = "name";
-    public final static String price = "price";
-    public final static String raw = "raw";
-    public final static String imported = "imported";
-    public final static String manufactured = "manufactured";
+	public static final String quantity = "quantity";
+    public static final String type = "type";
```

Comment:
Comments for variables ?

Reply:
Done with comments

Reply:
Do formatting right from next time in case of constants